scenario_player/main.py

```diff
@@ -169,71 +191,75 @@ def main(ctx, scenario_file, keystore_file, password, chains, data_path, auth, m
 
 
 @main.command(name="reclaim-eth")
+@click.option("--keystore-file", required=True, type=click.Path(exists=True, dir_okay=False))
+@click.password_option("--password", envvar="ACCOUNT_PASSWORD", required=True)
 @click.option(
     "--min-age",
     default=72,
     show_default=True,
     help="Minimum account non-usage age before reclaiming eth. In hours.",
 )
 @click.pass_obj
-def reclaim_eth(obj, min_age):
+def reclaim_eth(obj, min_age, password, keystore_file):
     from scenario_player.utils import reclaim_eth
 
-    reclaim_eth(min_age_hours=min_age, **obj)
+    configure_logging_for_subcommand(construct_log_file_name())
+
+    data_path = obj['data_path']
+    chain_rpc_urls = obj['chain_rpc_urls']
+    account = load_account_obj(keystore_file, password)
+
+    reclaim_eth(min_age_hours=min_age, chain_rpc_urls=chain_rpc_urls, data_path=data_path, account=account)
 
 
 @main.command('pack-logs')
 @click.option(
     '--target-dir', default=os.environ.get('HOME', './'), show_default=True,
     help='Target directory to pack logs to. Defaults to your home directory.'
 )
-@click.option(
-    '--scenario-names', 'scenario-names', required=True, multiple=True,
-    help='Scenarios to pack log files for.',
-)
 @click.option(
     '--pack-n-latest', default=1,
     help='Specify the max num of log history you would like to pack. Defaults to 1.'
          'Specifying 0 will pack all available logs for a scenario.',
 )
-@click.option(
-    '--raiden-dir', default=os.environ.get('HOME', '.') + '/.raiden',
-    help="Path to the raiden meta data dir. Defaults to ~/.raiden.",
-)
 @click.option('--post-to-rocket', default=True)
-def pack_logs(post_to_rocket, raiden_dir, pack_n_latest, scenario_names, target_dir):
-    raiden_dir = Path(raiden_dir)
-    if not raiden_dir.exists():
-        raise RuntimeError(f"{raiden_dir} does not exist!")
+@click.argument("scenario-file", type=click.File(), required=False)
+@click.pass_context
+def pack_logs(ctx, scenario_file, post_to_rocket, pack_n_latest, target_dir):
+    data_path = ctx.obj['data_path']
+
+    scenario_name = Path(scenario_file).stem
```

Comment:
This won't work `scenario_file` is a `click.File()` argument type which generates a File-like object. 
You can get at the path via the `.name` attribute but the better option would be to use `click.Path(exists=True, file_okay=True, dir_okay=False)` which directly returns the path as a string.

(Looks like the way we use `scenario_file` in what is now `run()` could also be changed to `Path` since we only ever use the `.name` attribute)